tests/framework/db/QueryTest.php

```diff
@@ -37,6 +37,16 @@ public function testSelect()
         $query->select('id, name');
         $query->addSelect('email');
         $this->assertEquals(['id', 'name', 'email'], $query->select);
+        
+        $query = new Query();
+        $query->select('name, lastname');
+        $query->addSelect('name');
+        $this->assertEquals(['name', 'lastname'], $query->select);
+        
+        $query = new Query();
+        $query->addSelect(['*', 'abc']);
```

Comment:
I'd test that selecting same column with two different aliases works well i.e.

```php
$query->addSelect(['field1 as a', 'field 1 as b']);
```

Reply:
Added